libs/output-mapping/src/Configuration/Adapter.php

```diff
@@ -72,49 +55,38 @@ public function getFileExtension()
     }
 
     /**
-     * @param $format
-     * @return $this
-     * @throws OutputOperationException
+     * @param self::FORMAT_YAML | self::FORMAT_JSON $format
      */
-    public function setFormat($format)
+    public function setFormat(string $format): Adapter
```

Comment:
nevim jestli z tohohle neco dedi, ale return type by mel byt spis `static`, je to nebliz `$this`